ansible_collections/arista/avd/roles/eos_designs/docs/tables/node-type-wan-configuration.md

```diff
@@ -97,6 +113,22 @@
           # Enable / Disable IPsec over HA path-group when HA is enabled.
           ipsec: <bool; default=True>
 
+          # Indicates the WAN HA interfaces selection mode.
+          # By default uplink interfaces in VRF default are used.
+          mode: <str; "uplinks" | "custom"; default="uplinks">
+
+        # PREVIEW: This key is currently not supported
+        #
+        # Local WAN HA interfaces
+        # Each list item supports range syntax that can be expanded into a list of interfaces.
+        wan_ha_interfaces:
```

Comment:
move this under wan_ha and check that it is deep merged
need to handle mixed scenarios
interfaces is used as a filtering mechanism (to be able to do partial uplinks + dedicated interfaces)